docs/resources/bedrock_blueprint.md

```diff
@@ -10,7 +9,35 @@ description: |-
 
 Definition of AWS::Bedrock::Blueprint Resource Type
 
-
+## Example Usage
+
+### Custom Blueprints for documents
+Create a custom blueprint for intelligent document processing with Bedrock Data Automation
+```terraform
+resource "awscc_bedrock_blueprint" "example" {
+  blueprint_name = "example-blueprint"
+  type           = "DOCUMENT"
+  schema = jsonencode({
+    "$schema" : "http://json-schema.org/draft-07/schema#",
+    "description" : "default",
+    "class" : "default",
+    "type" : "object",
+    "definitions" : {},
+    "properties" : {
+      "gross_pay_this_period" : {
+        "type" : "number",
+        "inferenceType" : "explicit",
+        "instruction" : "The gross pay for this pay period from the Earnings table"
+      },
+      "net_pay" : {
+        "type" : "number",
+        "inferenceType" : "explicit",
```

Comment:
(nit) : Could you add the tags for the resource ?

Reply:
@professional-ian The tags were added directly to the doc and so the `make docs` will override or fail on approval. Please add it to the .tf tile and run a `make docs`